dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/UserCommandKey.java

```diff
@@ -12,7 +12,11 @@
 import org.apache.hadoop.io.Text;
 import org.apache.hadoop.io.WritableComparable;
 
-
+/**
+ * <p>UserCommandKey is a {@link Writable} used as a composite key combining the user id and
```

Comment:
No need to start with a `<p>`. Only needed to make a subsequent paragraph break.